go/deploy/billaged/Makefile

```
@@ -0,0 +1,130 @@
+
+health: ## Check service health
+	@curl -s http://localhost:8081/health | jq . || echo "Health check failed"
+
+help: ## Show this help message
+	@echo ""
+	@echo "Billaged VM Usage Billing Service - Available targets:"
+	@echo ""
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2}'
+	@echo ""
+
+install: build create-user ## Install billaged binary and systemd service
```

checkmake (0.2.2)

[warning] Target body for "install" exceeds allowed length of 5 (7).

(maxbodylength)